Tensile/KernelWriter.py

```diff
@@ -1472,20 +1477,25 @@ def makeSubIterSchedule(self, kernel, localReadCode, iteration, pointerLWCode, p
             for j in range(instPerPack):
               iterCode.addCode(packItems.pop(0))
               curPackIdx += 1
-          if packItems:
+          # insert second packing code only if miLatencyLeft is large enough
+          if packItems and self.miLatencyLeft > 2:
             for j in range(instPerPack):
               iterCode.addCode(packItems.pop(0))
               curPackIdx += 1
           # since packed register need to wait 2 quad cycle to finish packing
           # we insert pack instruction if we can, or s_nop
+          count = 0 # count number of cycle for nop to insert
           while curPackIdx < numPack+2:
             if packItems:
               for j in range(instPerPack):
                 iterCode.addCode(packItems.pop(0))
                 curPackIdx += 1
             else:
-              iterCode.addInst("s_nop ","0","VALU packing writes to be consumed by matrix instruction")
+              count += 1
               curPackIdx += 1
+          if count:
+            # insert 1 nop instruction
+            iterCode.addInst("s_nop ",str(count - 1),"VALU packing writes to be consumed by matrix instruction")
```

Comment:
This should probably be in KernelWriterAssembly, base class is supposed to abstract specifics of hip vs assembly code

Reply:
I just changed the count part only and it was already there before my change.
Do you want me to change that in this PR?
It is not directly related ty my change.

Reply:
Oh yea I see that was there before. No need to change in this PR, but maybe file a ticket so someone can update it later.